codespell_lib/data/dictionary.txt

```diff
@@ -16564,7 +16577,8 @@ lpatform->platform
 lsat->last
 lsit->list
 lsits->lists
-lukid->likud
+lukid->lucid, likud,
+lumination->lamination, illumination,
```

Comment:
Hmmm? https://www.collinsdictionary.com/dictionary/english/lumination

Reply:
All Dictionaries I have looked it up, don't know it.
https://dictionary.cambridge.org/de/spellcheck/englisch/?q=lumination

I will have a look on some physics papers later, if it was/is used by people.

Reply:
I think most people, who use it, want something like luminance?!

Reply:
Chuck it in rare:
http://app.aspell.net/lookup?dict=en_US&words=lumination%0D%0Aillumination